blocks/init/src/Blocks/components/embed/embed-style.scss

```diff
@@ -0,0 +1,15 @@
+.embed {
+	position: relative;
+	width: 100%;
+	overflow: hidden;
+	padding-top: 56.25%;
+
+	&__iframe {
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		border: 0;
+	}
+}
```

Comment:
Please rework this so it doesn't use the old aspect ratio hack, we've had `aspect-ratio` in all browsers for 3 years now...